budgettool/matplot_view.py

```diff
@@ -0,0 +1,14 @@
+"""Plot graph of forecast."""
+
+def plot_forecast(forecast):
+    """Plot a graph of the forecast."""
+    import matplotlib.pyplot as plt
```

Comment:
https://www.python.org/dev/peps/pep-0008/#imports

"""
Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants.
"""